installing/installing_alibaba/manually-creating-alibaba-ram.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: ASSEMBLY
+[id="manually-creating-alibaba-ram"]
+= Creating the required Alibaba Cloud resources
+include::modules/common-attributes.adoc[]
+:context: manually-creating-alibaba-ram
+
+toc::[]
+
+Before you install OpenShift Container Platform, you must use the Alibaba Cloud console to create a Resource Access Management (RAM) user that has sufficient permissions to install {product-title} into your Alibaba Cloud. This user must also have permissions to create new RAM users. You can also configure and use the `ccoctl` tool to create new credentials for the {product-title} components with the permissions that they require.
+
+:FeatureName: Alibaba Cloud on {product-title}
+include::snippets/technology-preview.adoc[]
+
+//Task part 1: Manually creating the required RAM user
+include::modules/manually-creating-alibaba-ram-user.adoc[leveloffset=+1]
+
+//Task part 2: Configuring the Cloud Credential Operator utility
+include::modules/cco-ccoctl-configuring.adoc[leveloffset=+1]
+
+//Task part 3: Creating AWS resources with a single command
+include::modules/cco-ccoctl-creating-at-once.adoc[leveloffset=+1]
+
+== Next steps
+
+* Install a cluster on Alibaba Cloud infrastructure that is provisioned by the {product-title} installation program, by using one of the following methods:
+
+** **xref:../../installing/installing_alibaba/installing-alibaba-default.adoc#installing-alibaba-default[Installing a cluster quickly on Alibaba Cloud]**: You can install {product-title} on Alibaba Cloud infrastructure that is provisioned by the {product-title} installation program. You can install a cluster quickly by using the default configuration options.
+
+** **xref:../../installing/installing_alibaba/installing-alibaba-customizations.adoc#installing-alibaba-customizations[Installing a customized cluster on Alibaba Cloud]**: You can install a customized cluster on Alibaba Cloud infrastructure that the installation program provisions. The installation program allows for some customization to be applied at the installation stage. Many other customization options are available xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[post-installation].
```

Comment:
Same here. You can probably remove: "You can install a customized cluster on Alibaba Cloud infrastructure that the installation program provisions."